api/server/server.go

```diff
@@ -1631,7 +1639,27 @@ func ServeApi(job *engine.Job) error {
 	return nil
 }
 
-func toBool(s string) bool {
-	s = strings.ToLower(strings.TrimSpace(s))
-	return !(s == "" || s == "0" || s == "no" || s == "false" || s == "none")
+func containerCommit(eng *engine.Engine, name string, c *daemon.ContainerCommitConfig) (string, error) {
```

Comment:
Any reason this is here instead of in the builder dir?  It seems builder specific.  Is it part of the api?